.github/scala-steward.conf

```diff
@@ -1,9 +1,18 @@
 pullRequests.frequency = "@monthly"
 
-commits.message = "${artifactName} ${nextVersion} (was ${currentVersion})"
+commits.message = "[2.9.x] ${artifactName} ${nextVersion} (was ${currentVersion})"
 
 pullRequests.grouping = [
-  { name = "patches", "title" = "Patch updates", "filter" = [{"version" = "patch"}] }
+  { name = "patches", "title" = "[2.9.x] Patch updates", "filter" = [{"version" = "patch"}] }
+]
+
+updates.pin = [
+  // Play 2.9
+  { groupId = "com.typesafe.play", artifactId = "sbt-plugin", version = "2.9" },
```

Comment:
Ops, too fast, seems the version need to have a `.` as suffix: https://github.com/scala-steward-org/scala-steward/blob/main/docs/faq.md#how-can-version-updates-be-controlled

So here it should be `2.9.` not just `2.9`...

Reply:
- #522

Reply:
Hmm 🤔 Will this config not work without ending `.`? Or it's just possible to get PR with bump to `2.90.0`? 😄

Reply:
To be honest I did not check if it is working or not, I just saw that in the docs and also in our other repos we always add `.` at the end, so I think if we know this is working we probably keep it that way. Maybe it also works without the dot... I don't know.

Reply:
Ok. Sorry for that.